### PR TITLE
Task Aliases

### PR DESCRIPTION
**Pull request by splatteredbits**
If I want to change a task's name, there should be a way to make the old name usable with the new name and warn users to use the task's new name.

### CODE REVIEW

File: Whiskey/Functions/Invoke-WhiskeyTask.ps1 [hunk 1]
```diff
@@ -115,6 +115,15 @@ function Invoke-WhiskeyTask
 
     $task = $knownTasks | Where-Object { $_.Name -eq $Name }
 
+    if( -not $task )
+    {
+        $task = $knownTasks | Where-Object { $_.Aliases -contains $Name }
```
Comment (DecoyJoe) [inline Whiskey/Functions/Invoke-WhiskeyTask.ps1]: Should we also check to make sure this resolved to a single task? What if there are 2 more tasks that accidentally share a common alias?

File: appveyor.yml [hunk 2]
```diff
@@ -17,11 +17,14 @@ environment:
...
 build_script:
+#- sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e -
 - pwsh: >-
     # $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
     
     # On Linux, progress bars still show when running on the build server and take *ages* to render and they screw up the displayed output.
-    $ProgressPreference = 'SilentlyContinue'
+    $Global:ProgressPreference = 'SilentlyContinue'
+    
+    Write-Verbose -Message ([Environment]::UserInteractive) -Verbose
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply appveyor.yml]: Removed the `Write-Verbose` statement.

File: Whiskey/Functions/Invoke-WhiskeyTask.ps1 (hunk 1 above)
Comment (splatteredbits) [reply Whiskey/Functions/Invoke-WhiskeyTask.ps1]: Fixed. I also added a test and check for duplicate task names.

File: Whiskey/Functions/Invoke-WhiskeyTask.ps1 [hunk 3]
```diff
@@ -118,6 +118,12 @@ function Invoke-WhiskeyTask
     if( -not $task )
     {
         $task = $knownTasks | Where-Object { $_.Aliases -contains $Name }
+        $taskCount = ($task | Measure-Object).Count
+        if( $taskCount -gt 1 )
+        {
+            Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Found {0} tasks with alias "{1}". Please update to use one of these task names: {3}.' -f $taskCount,$Name,$TaskContext.ConfigurationPath.FullName,(($task | Select-Object -ExpandProperty 'Name') -join ', '))
```
Comment (DecoyJoe) [inline Whiskey/Functions/Invoke-WhiskeyTask.ps1]: Number of tokens doesn't match the number of string format parameters.
